target/linux/ramips/image/mt7621.mk

```diff
@@ -709,6 +709,16 @@ define Device/xiaomi_mir3p
 endef
 TARGET_DEVICES += xiaomi_mir3p
 
+define Device/xiaoyu_xy-c5
+  MTK_SOC := mt7621
+  IMAGE_SIZE := 32448k
+  DEVICE_VENDOR := XiaoYu
+  DEVICE_MODEL := XY-C5
+  DEVICE_PACKAGES := \
+	kmod-ata-core kmod-ata-ahci kmod-usb3
```

Comment:
You can remove the `kmod-ata-core`package inclusion, as it's a dependency of `kmod-ata-ahci`.

To make the formatting nice, you can remove the line-break for `DEVICE_PACKAGES` afterwards :+1:

Reply:
scroll up this file and I found many kmod-ata-core and kmod-ata-ahci together.w hy?

Reply:
> scroll up this file and I found many kmod-ata-core and kmod-ata-ahci together.w hy?

Because there nobody checked for dependencies ...

Reply:
I start this job modified from PBR-M1 and many issue come from it :)